package.json

```diff
@@ -1,16 +1,16 @@
 {
   "dependencies": {
-    "alex": "~2.0.1",
-    "autoprefixer": "~6.3.4", 
-    "bootlint": "~0.14.2",
-    "coffeelint": "~1.14.2",
-    "complexity-report": "~2.0.0-alpha",
-    "csslint": "~0.10.0",
-    "dockerfile_lint": "~0.0.9",
-    "eslint": "~2.3.0",
-    "jshint": "~2.9.1",
-    "postcss-cli": "~2.5.1",
-    "remark": "~3.2.3",
-    "tslint": "~3.5.0"
+    "alex": "~2",
+    "autoprefixer": "~6",
```

Comment:
Erm, I don';t think this is that you want.

Im not even sure what this means, but i'd guess that this means:
`~6` => `6` or `x` where `x` > 6

You want to do `~6.3`, which should mean `6.3` or `6.x` where `x` > 3